generators/app/templates/src/main/java/package/web/rest/_JaversEntityAuditResource.java

```diff
@@ -66,13 +66,13 @@ public JaversEntityAuditResource(Javers javers) {
     @RequestMapping(value = "/audits/entity/changes",
         method = RequestMethod.GET,
         produces = MediaType.APPLICATION_JSON_VALUE)
-    @Timed
     @Secured(AuthoritiesConstants.ADMIN)
     public ResponseEntity<List<EntityAuditEvent>> getChanges(@RequestParam(value = "entityType") String entityType,
-                                                             @RequestParam(value = "limit") int limit)
+                                                             @RequestParam(value = "limit") int limit,
+                                                             @RequestParam MultiValueMap<String, String> queryParams,
+                                                             UriComponentsBuilder uriBuilder)
         throws URISyntaxException, ClassNotFoundException {
         log.debug("REST request to get a page of EntityAuditEvents");
-        Pageable pageRequest = createPageRequest(limit);
```

Comment:
This is never used. So I've also removed the function `createPageRequest(int size)`